v3/src/components/graph/adornments/movable-line.tsx

```diff
@@ -56,6 +56,23 @@
     xSubAxesCount = layout.getAxisMultiScale('bottom')?.repetitions ?? 1,
     ySubAxesCount = layout.getAxisMultiScale('left')?.repetitions ?? 1
 
+  const setEquationContainerClass = useCallback(() => {
```

Comment:
Similar comment for setEquationContainerSelector possibly better named just equationContainerSelector?

Reply:
I concur with @bfinzer -- these functions are computing/returning a value, not `set`ting something (e.g. state).